Calls ldconfig which is not available and remove -march=native because it's not
portable across platforms.

Upstream: no, upstream author isn't cooperative.
diff --git a/source/Makefile b/source/Makefile
index c7aa3b3..b2140b9 100644
--- a/source/Makefile
+++ b/source/Makefile
@@ -31,11 +31,9 @@ VERSION = $(MAJVERS).$(MINVERS)
 DISTDIR = zita-resampler-$(VERSION)
 
 CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
-CPPFLAGS += -DENABLE_SSE2
 CPPFLAGS += -DZITA_RESAMPLER_MAJOR_VERSION=$(MAJVERS)
 CPPFLAGS += -DZITA_RESAMPLER_MINOR_VERSION=$(MINVERS)
-CXXFLAGS += -Wall -fPIC -O2 -ffast-math
-CXXFLAGS += -march=native
+CXXFLAGS += -Wall -fPIC -O2
 LDFLAGS += 
 LDLIBS +=
 
